types/fake-response/index.d.ts

```diff
@@ -0,0 +1,214 @@
+     * @example
+     * const {FakeResponse} = require("fake-response");
+     * const fakeResponse = new FakeResponse(db);
+     * const app = fakeResponse.createExpressApp(); // creates and returns the express app.
+     * @link https://r35007.github.io/Fake-Response/#createexpressapp - For further info pls visit this ReadMe
+     */
+    createExpressApp: () => express.Application;
+    /**
+     * This function starts the express server.
+     * Please make sure you create the express app before starting the server
+     * @example
+     * const {FakeResponse} = require("fake-response");
+     * const fakeResponse = new FakeResponse(db);
+     * const app = fakeResponse.createExpressApp();
+     * fakeResponse.startServer(3000); // the port is an optional param
+     * @link https://github.com/R35007/Fake-Response#startserver - For further info pls visit this ReadMe
+     */
+    startServer: (port?: number) => Promise<Server>;
+    /**
+     * This function stops the express server
+     * @example
+     * const {FakeResponse} = require("fake-response");
+     * const fakeResponse = new FakeResponse(db);
+     * const isStopped = fakeResponse.stopServer(); // make sure the server is already started
+     * @link https://github.com/R35007/Fake-Response#stopserver - For further info pls visit this ReadMe
+     */
+    stopServer: () => Promise<boolean>;
+    /**
+     * This function helps to generate a route before starting the server
+     * @example
+     * const {FakeResponse} = require("fake-response");
+     * const fakeResponse = new FakeResponse(db);
+     * const result = fakeResponse.loadResources();
+     * @link https://github.com/R35007/Fake-Response#loadresources - For further info pls visit this ReadMe
+     */
+    loadResources: () => Promise<RouteResult[]>;
+    /**
+     * This function helps to create a route explicitly
+     * @example
+     * const {FakeResponse} = require("fake-response");
+     * const fakeResponse = new FakeResponse(db);
+     *
+     *const newResponse = {
+          value : "New response"
+     }
+     * fakeResponse.createRoute('/newRoute',newResponse);
+     * @link https://github.com/R35007/Fake-Response#createroute - For further info pls visit this ReadMe
+     */
+    createRoute: (data: any, route: string, dataType?: string, middleware?: Middleware, delay?: number) => void;
+    /**
+     * This function helps to create a default route explicitly.
+     * This creates the following routes.
+     * http://localhost:3000/
+     * http://localhost:3000/db
+     * http://localhost:3000/routesList
+     *
+     * Note : The port is assigned from the config
+     * @example
+     * const {FakeResponse} = require("fake-response");
+     * const fakeResponse = new FakeResponse(db);
+     * fakeResponse.createDefaultRoutes();
+     * @link https://github.com/R35007/Fake-Response#createdefaultroutes - For further info pls visit this ReadMe
+     **/
```

Comment:
```suggestion
     */
```
same, `npm run lint fake-reponse` catch this nicely